app/services/escort_issuer.rb

```diff
@@ -38,7 +38,7 @@ def issued_per_document
 
   def generate_per_document
     pdf = PdfGenerator.new(escort).call
-    file = Tempfile.new('per')
+    file = Tempfile.new(['per', '.pdf'])
```

Comment:
ideally this would actually use a filename we wanted
rather than a random one.

But decided that was out of scope of this quick fix.